models/users.js

```diff
@@ -572,6 +572,28 @@ const fetchAllUsers = async () => {
   return users;
 };
 
+const archiveUserIfNotInDiscord = async () => {
```

Comment:
For batch operations like this, it's usually beneficial to include additional metrics in the response, such as the number of users whose "in discord" field was updated. This way, it informs and assures the user that the operation was successful.